### ISSUE TITLE
LibGSL.gsl_vector_add not working

### ISSUE THREAD

**Issue by dylandy** [dylandy/crystal-gsl#1]
always return 0.

sample:
```crystal
LibGSL.gsl_vector_add([1,2,3].to_vector.getPointer , [1,2,3].to_vector.getPointer)
=> 0
```

**Comment by jlebray** [comment]
What number should we be getting ? [The doc](https://www.gnu.org/software/gsl/manual/html_node/Vector-operations.html) says it modifies the vector in place :confused:

**Comment by dylandy** [comment]
The doc said
```
The result a_i  <-  a_i + b_i is stored in a and b remains unchanged.
```
I finally understand....

![test](https://cloud.githubusercontent.com/assets/2987161/20917635/8cb4fae8-bbcd-11e6-84c7-f4876c0428b5.png)

**Comment by dylandy** [comment]
I thought it should be something like this

![test](https://cloud.githubusercontent.com/assets/2987161/20917791/755c9d14-bbce-11e6-96be-13aa987ba607.png)

overwrite the original vector is ambiguous. Orz

**Comment by dylandy** [comment]
Fixed in further commit  c8ad8ab
